### PR TITLE
[138] Product Page using HTML

### PR DESCRIPTION
**Pull request by Nelyne**

### CODE REVIEW

**Comment by Nelyne** [comment]
Annotation 2020-11-30 160250
Annotation 2020-11-30 160510

File: submissions/nkeirukaEnendu/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+           </ul>
+       </div>
+   </nav>
+   <div>
+       <img src="./img - Copy/Rectangle 11.png" alt="">
+       <img src="./img - Copy/Rectangle 756.png" alt="">
+   </div>
+   <p><strong>Amount: N15,000,000</strong></p>
+   <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Repellat, fugiat corporis quam exercitationem officia enim dolor a obcaecati cumque aperiam ipsum maxime explicabo velit eius ipsa tempora, illum in laboriosam sunt inventore, nemo saepe. Sunt expedita eveniet mollitia culpa debitis doloribus, tempora voluptate consectetur voluptatum asperiores incidunt itaque veritatis odit.</p>
+
```
Comment (Ireti203) [inline submissions/nkeirukaEnendu/HTML/index.html:27]: Remove whitespace

File: submissions/nkeirukaEnendu/HTML/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   <h2>Also Available</h2>
+   <ol>
+       <li>Italian Designed Office Chairs</li>
+       <li>American Imported Parlour Sittings</li>
+       <li>Handcrafted Throw Pillows</li>
+       <li>Oak Tree Center Table</li>
+   </ol>
+
```
Comment (Ireti203) [inline submissions/nkeirukaEnendu/HTML/index.html:35]: Remove whitespace

File: submissions/nkeirukaEnendu/HTML/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   <span>&copy; 2020</span>
+
```
Comment (Ireti203) [inline submissions/nkeirukaEnendu/HTML/index.html:37]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**